.husky/commit-msg

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
```

Comment:
Should probably remove this line too (I see you removed it in the other hook)

Reply:
Kept it cause we have `$1` (shell syntax) and having the hashbang allows the IDE to understand that it's shell script

The husky shell script execution is removed as suggested by the migration guide

Reply:
Then it should be brought back in the other shell script

Reply:
- [x] nevermind, removed the hashbang in 9bf20c6